helm/examples/azure/hpcc-azurefile/templates/NOTES.txt

```diff
@@ -16,3 +17,6 @@ storage:
 
   dataStorage:
     existingClaim: "{{ $dataPvcName }}"
+
+  sashaStorage:
```

Comment:
I add "sashaStorage" in schema and values.yaml. Also add pvc and storageclass for it. Thought I need shared sasha pvc for sasha/wu-archiver and sasha/dfuwu-archiver storage

Reply:
OK I will flow localfile pattern to make one pair of pvc and sc yaml to handle all the planes

Reply:
you do need a pvc for wu-archiver and dfuarchiver, but a sashsStorage in the values file fed to the hpcc chart would be invalid, and indeed you do not have one in the changed values-*-azurefile.yaml files.

The output helm install hpcc-azurefile (and same for hpcc-localfile) is supposed to be able to be captured and used instead of the example values-retained-azurefile.yaml file, e.g.:
```
helm install azstorage examples/azure/hpcc-azurefile/ | grep -A100 '^storage:' > mystorage.yaml
helm install myhpcc hpcc/ .... -f mystorage.yaml 
```

values-retained-azurefile.yaml is supposed to an example of that, so should exactly match the output from the above if the install name is the same.

Reply:
> OK I will flow localfile pattern to make one pair of pvc and sc yaml to handle all the planes

@xwang2713 - could get this merged, then work on unifying using planes in new JIRA/PR.